cmd/ui/src/App.tsx

```diff
@@ -108,10 +111,20 @@ const Inner: React.FC = () => {
         }
     }, [dispatch, authState.isInitialized]);
 
-    // prefetch feature flags
+    // initialize theme
     useEffect(() => {
-        queryClient.prefetchQuery(featureFlagKeys.all, getFeatureFlags);
```

Comment:
This just warms the cache up with the feature flag response. But the cache will get populated anyway now that we are requesting feature flags from this level.